packages/picasso-query-builder/src/QueryBuilder/story/index.jsx

```diff
@@ -38,6 +38,11 @@ page
     description: 'Customized header',
   })
 
+  .addExample('QueryBuilder/story/FieldDescription.example.tsx', {
+    title: 'Field Description',
+    description: `Hover over 'specificFieldName' to display field description in a tooltip`,
```

Comment:
When I said `specificFieldName` i was assuming you would add the actual field name like:

```suggestion
    description: `Hover over 'First  name' field in the field selector dropdown to display field description in a tooltip`,
```